support/CMakeLists.txt

```diff
@@ -44,9 +44,14 @@ add_library(support
 
 add_library(sleigh::support ALIAS support)
 
-target_link_libraries(support PUBLIC
-  sleigh::sleigh_settings
+target_compile_features(support PRIVATE
```

Comment:
I didn't bother making an extra settings lib like we do with `sleigh_settings` since we're only using it for the support lib.

Reply:
Ah, actually I need to make an extra settings lib anyway. `sleigh-lift` will need it.